### PR TITLE
use mcr.microsoft.com for ubuntu

### PR DESCRIPTION
**Pull request by TravisEz13**

### CODE REVIEW

File: tools/releaseBuild/images/Ubuntu16.04/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM microsoft/powershell:ubuntu16.04
+FROM mcr.microsoft.com/powershell:ubuntu16.04
```
Comment (andschwa) [inline tools/releaseBuild/images/Ubuntu16.04/Dockerfile:1]: Is it possible to verify the validity of this URL through a web UI? I don't have Docker setup anywhere, and going to mcr.microsoft.com redirects to a GitHub repository.

Comment (TravisEz13) [reply tools/releaseBuild/images/Ubuntu16.04/Dockerfile:1]: It's not a url so no.

Comment (TravisEz13) [reply tools/releaseBuild/images/Ubuntu16.04/Dockerfile:1]: image
```suggestion
FROM mcr.microsoft.com/powershell:ubuntu-16.04
```

Comment (andschwa) [reply tools/releaseBuild/images/Ubuntu16.04/Dockerfile]: Thanks 😂

**Review by andschwa — APPROVED** [review]

**Pull request merged**